### ISSUE TITLE
Automations mod problems

### ISSUE THREAD

**Issue by CV514** [Kenan2000/Bright-Nights-Kenan-Mod-Pack#10]
```
ERROR : src/requirements.cpp:423 [void component::check_consistency(const string&) const] bot_utilibot_base in inline_recipe_bot_utility_w_rope_to_thread is not a valid item template
ERROR : src/requirements.cpp:423 [void component::check_consistency(const string&) const] bot_utilibot_butler in inline_recipe_bot_butler_w_scramble is not a valid item template
ERROR : src/requirements.cpp:423 [void component::check_consistency(const string&) const] bot_utilibot_butler in inline_recipe_bot_butler_w_scramble_5 is not a valid item template
ERROR : src/requirements.cpp:423 [void component::check_consistency(const string&) const] bot_utilibot_base in inline_recipe_bot_utility_w_rope_to_thread_5 is not a valid item template
ERROR : src/requirements.cpp:423 [void component::check_consistency(const string&) const] bot_utilibot_base in inline_recipe_bot_utility_w_rag_to_socks is not a valid item template
ERROR : src/requirements.cpp:423 [void component::check_consistency(const string&) const] bot_utilibot_base in inline_recipe_bot_utility_w_rag_to_socks_5 is not a valid item template
ERROR : src/requirements.cpp:423 [void component::check_consistency(const string&) const] bot_utilibot_base in inline_recipe_bot_utility_w_rag_to_briefs is not a valid item template
ERROR : src/requirements.cpp:423 [void component::check_consistency(const string&) const] bot_utilibot_base in inline_recipe_bot_utility_w_rag_to_briefs_5 is not a valid item template
ERROR : src/requirements.cpp:423 [void component::check_consistency(const string&) const] bot_utilibot_base in inline_uncraft_bot_utility_f_rope_to_thread is not a valid item template
ERROR : src/requirements.cpp:423 [void component::check_consistency(const string&) const] bot_utilibot_butler in inline_uncraft_bot_butler_f_scramble is not a valid item template
ERROR : src/requirements.cpp:423 [void component::check_consistency(const string&) const] bot_utilibot_butler in inline_uncraft_bot_butler_f_scramble_5 is not a valid item template
ERROR : src/requirements.cpp:423 [void component::check_consistency(const string&) const] bot_utilibot_base in inline_uncraft_bot_utility_f_rope_to_thread_5 is not a valid item template
ERROR : src/requirements.cpp:423 [void component::check_consistency(const string&) const] bot_utilibot_base in inline_uncraft_bot_utility_f_rag_to_socks is not a valid item template
ERROR : src/requirements.cpp:423 [void component::check_consistency(const string&) const] bot_utilibot_base in inline_uncraft_bot_utility_f_rag_to_socks_5 is not a valid item template
ERROR : src/requirements.cpp:423 [void component::check_consistency(const string&) const] bot_utilibot_base in inline_uncraft_bot_utility_f_rag_to_briefs is not a valid item template
ERROR : src/requirements.cpp:423 [void component::check_consistency(const string&) const] bot_utilibot_base in inline_uncraft_bot_utility_f_rag_to_briefs_5 is not a valid item template
```

Mod dependencies are defined as:
`"dependencies": [ "dda", "aftershock", "modular_turrets_no_blacklist", "Salvaged_Robots" ]`

So, I've looked into `bot_utilibot_base` and it seems like this is not some defined item, but an abstract for other bots in Salvaged Robots mod. Both `items.json` and `recipe.json` define it not as "id", but as "abstract", and because this is an abstract object, it does not appear in the game as a real item part you can use in recipes and receive from uncraft while Automations mod is trying to do so.

Proposition:
Review this mod utility and functions in a whole since it looks like something that can be merged into Salvaged Robots mod and/or done entirely different. For now, as I understand, this thing (even if it was working) is just "transforming" one bot type to another, which will provide necessary "crafted" items in uncraft process as a "craft result". Way to clunky and inconvenient, or I miss something here.

**Comment by Kenan2000** [comment]
I honestly wish I could delete this mod altogether since I'm literally unaware how to handle this issue, I mostly try to maintain what I'm good at, in this case, this problem is too convoluted for me to tackle it, I'm still working on fixing CDDA Modpack - plenty of errors to handle, then continuing to work on BL9 expansion as well, so no idea /:

**Comment by CV514** [comment]
No problem then, I'll look into it and try to assemble some PR to incorporate its functions into Salvage Robots. Perhaps some time next week.

**Comment by CV514** [comment]
Resolved as from 538fd0cd599771225e345a54ad7f056a4978351b (functionality update) and 603464b7d4ee27430b84f3625cc4644f4510fb53 (removal of Automations mod)
